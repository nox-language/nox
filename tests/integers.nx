let var n := 10
    var minus_n := -n
    var addition := n + 2 + minus_n
    var result := minus_n / addition
in (
    printi(10 / 2);
    printi(minus_n);
    printi(addition);
    printi(result);
    printi(200 - n * minus_n)
)
end
