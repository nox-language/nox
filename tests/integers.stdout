5
-10
2
-5
300
